### PR TITLE
fix issue#144

### PR DESCRIPTION
**Pull request by hawkeoni**
Found 2 TODO's with replace txt and cleaned the code.

### CODE REVIEW

**Comment by hawkeoni** [comment]
this is about https://github.com/datenanfragen/website/issues/144

File: src/Utility/requests.js [hunk 1]
```diff
@@ -71,9 +71,8 @@ export function fetchTemplate(locale, request_type, company = null, suffix = 'de
         : request_type + (suffix ? '-' + suffix : '');
 
     if (!Object.keys(I18N_DEFINITION_REQUESTS).includes(locale)) locale = LOCALE;
-    // TODO: Once this is merged, remove the `.txt` in custom template declarations in the DB. That is just silly.
     const template_url =
-        BASE_URL + 'templates/' + (locale || LOCALE) + '/' + (template ? template.replace(/\.txt$/, '') + '.txt' : '');
+        BASE_URL + 'templates/' + (locale || LOCALE) + '/' + (template ? template + '.txt' : '');
```
[the remark this answers is not in the document]
Comment (hawkeoni) [reply src/Utility/requests.js]: Sure, I'lll remove it right away!

File: src/Utility/requests.js [hunk 2]
```diff
@@ -72,7 +72,7 @@ export function fetchTemplate(locale, request_type, company = null, suffix = 'de
 
     if (!Object.keys(I18N_DEFINITION_REQUESTS).includes(locale)) locale = LOCALE;
     const template_url =
-        BASE_URL + 'templates/' + (locale || LOCALE) + '/' + (template ? template + '.txt' : '');
+        BASE_URL + 'templates/' + (locale || LOCALE) + '/' + template;
```
Comment (zner0L) [inline src/Utility/requests.js]: We still need to add the `.txt` at the end.

Comment (hawkeoni) [reply src/Utility/requests.js]: my bad

**Pull request merged**